com.unity.render-pipelines.high-definition/Editor/AssetProcessors/MaterialPostProcessor.cs

```diff
@@ -334,6 +334,8 @@ static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAsse
                 // Materials (.mat) post processing:
 
                 var material = (Material)AssetDatabase.LoadAssetAtPath(asset, typeof(Material));
+                if (material == null)
+                    continue;
```

Comment:
I wonder if we should threat this an exception or log error. It feels risky we silently fail to process a material.

Reply:
Fair point, I'm not sure. As I don't have a repro case, I'm working under the impression that these are not really materials, but some other assets that happen to have a .mat extension. But I'm not sure if that's even allowed or makes sense in any valid use case. Does unity have strict 1:1 mapping between asset file extensions and asset types? Perhaps this needs some deeper investigation...